src/EVEMon/SkillPlanner/PlanEditorControl.Designer.cs

```diff
@@ -599,6 +601,18 @@ private void InitializeComponent()
             this.lblPartiallyTrained.Text = "Partially Trained";
             this.lblPartiallyTrained.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
             // 
+            // lblOmegaOnly
+            // 
+            this.lblOmegaOnly.AutoSize = true;
+            this.lblOmegaOnly.BackColor = System.Drawing.Color.Gold;
+            this.lblOmegaOnly.BorderStyle = System.Windows.Forms.BorderStyle.FixedSingle;
+            this.lblOmegaOnly.ForeColor = System.Drawing.SystemColors.ControlText;
+            this.lblOmegaOnly.Margin = new System.Windows.Forms.Padding(0, 0, 4, 0);
+            this.lblOmegaOnly.Name = "lblOmegaOnly";
+            this.lblOmegaOnly.TabIndex = 38;
```

Comment:
Just picked the next unused TabIndex. Doesn't seem to matter as you don't tab around these labels anyway. Also left off Size and Location as they're determined automatically.